preview-src/scroll-sync.ts

```diff
@@ -96,11 +96,13 @@ export function scrollToRevealSourceLine (line: number) {
       // Between two elements. Go to percentage offset between them.
       const betweenProgress = (line - previous.line) / (next.line - previous.line)
       const elementOffset = next.element.getBoundingClientRect().top - previousTop
-      scrollTo = previousTop + betweenProgress * elementOffset
+      scrollTo = window.scrollY + previousTop + betweenProgress * elementOffset
```

Comment:
Probably a rebase issue? This change was part of another pull request.

Reply:
True -- I'm not sure why this occurred. I did do a rebase which succeeded without any conflicts... "A git thing" I suppose.

Reply:
Try to fetch origin and rebase on `origin/master`. I can see that you did a merge which is not recommended because it creates a merge commit that won't play nice with the next rebases.

```
Merge branch 'issue-99-link-provider-2' of (...)
```